File: Models/Intermediation.cs
using System;
using System.Collections.Generic;

namespace PremiumcontactMaster.Models
{
    public partial class Intermediation
    {
        public int Idintermediation { get; set; }
        public int? IdlisteService { get; set; }
        public int? IdlisteSousService { get; set; }
        public DateTime? DateIntermediation { get; set; }
        public string Intermediation1 { get; set; }
        public string Lieu { get; set; }
        public string Description { get; set; }
        public bool? ChoixdeRdv { get; set; }
        public int? Iddocument { get; set; }
        public short? IdenquetePageWeb { get; set; }
        public int? IdselectionIntermediation { get; set; }
        public int? IdselectionListePlage { get; set; }
        public int? IdselectionListeIndisponibilite { get; set; }
        public string IntituleTypeVendeur { get; set; }
        public short? IdenqueteVendeur { get; set; }
        public int? IdabonnementVendeur { get; set; }
        public int IdutilisateurVendeur { get; set; }
        public string IntituleTypeAcheteur { get; set; }
        public short? IdenqueteAcheteur { get; set; }
        public short? IdenqueteAcheteurBesoin { get; set; }
        public int? IdabonnementAcheteur { get; set; }
        public int? IdutilisateurAcheteur { get; set; }
        public byte Idstatut { get; set; }
        public string SiteWebIntermediation { get; set; }
        public string IntermediationParametres { get; set; }
        public DateTime? OuvertureDateChoix { get; set; }
        public string OuvertureHeureChoix { get; set; }
        public DateTime? FermetureTheoriqueDateChoix { get; set; }
        public string FermetureTheoriqueHeureChoix { get; set; }
        public DateTime? FermetureDateChoix { get; set; }
        public string FermetureHeureChoix { get; set; }
    }
}

File: ClientApp/src/app/Models/Intermediation.ts

interface Intermediation {
    idintermediation: number;
    idlisteService: number;
    idlisteSousService: number;
    dateIntermediation: Date;
    intermediation1: string;
    lieu: string;
    description: string;
    choixdeRdv: boolean;
    iddocument: number;
    idenquetePageWeb: number;
    idselectionIntermediation: number;
    idselectionListePlage: number;
    idselectionListeIndisponibilite: number;
    intituleTypeVendeur: string;
    idenqueteVendeur: number;
    idabonnementVendeur: number;
    idutilisateurVendeur: number;
    intituleTypeAcheteur: string;
    idenqueteAcheteur: number;
    idenqueteAcheteurBesoin: number;
    idabonnementAcheteur: number;
    idutilisateurAcheteur: number;
    idstatut: ByteString;
    siteWebIntermediation ?: string;
    intermediationParametres: string;
    ouvertureDateChoix: Date;
    ouvertureHeureChoix: string;
    fermetureTheoriqueDateChoix: Date;
    fermetureTheoriqueHeureChoix: string;
    fermetureDateChoix: Date;
    fermetureHeureChoix: string;
}
interface ObjectResult_Intermediation {
    description: string;
    errorCode: string;
    result: Intermediation;
}


